### DISCUSSION TITLE
DB structure initialization

### DISCUSSION THREAD

**Discussion by PepaBosnis** [mealie-recipes/mealie#4463] [Q&A]
Hi .. I haven't found anywhere how to properly populate the database with the necessary structure ... the instructions I found neither of them works. Could someone advise me .. the necessary commands.
For clarification.. I am trying to get this up and running on win server 2016 standard + IIS 10, the build was successful, now I need the DB structure.

**Comment by Kuchenpirat** [comment]
Please note that we currently [only support deployments via Docker](https://docs.mealie.io/documentation/getting-started/installation/installation-checklist/#pre-work) and advise against running mealie in other scenarios if you don't know exactly what you are doing. 


That being said you probably want to have alembic run all the migrations to get up to the latest state. 
This would normally happen automatically on application startup. Without knowing your exact setup i would advise you have a look at the alembic folder and try to
